### PR TITLE
Add ability to encrypt splunk traffic

### PR DESCRIPTION
**Pull request by maxrothman**
Implemented by https://github.com/edx-ops/mckinsey-secure/pull/170

### CODE REVIEW

File: playbooks/roles/splunk-server/tasks/main.yml [hunk 1]
```diff
@@ -27,6 +27,26 @@
...
+    content: "{{ item.1 }}"
+    owner: splunk
+    group: splunk
+    mode: "0400"
+  when: "{{ SPLUNK_SSL_CERT }}"
+  with_together:
+    - [forwarder.pem, cacert.pem]
+    - ["{{ SPLUNK_SSL_CERT }}", "{{ SPLUNK_ROOT_CA }}"]
+
+- name: Create inputs configuration
```
Comment (fredsmith) [inline playbooks/roles/splunk-server/tasks/main.yml:52]: Is this written for every splunk server installation?  What happens when SPLUNK_SSL_CERT is !!null?

Comment (maxrothman) [reply playbooks/roles/splunk-server/tasks/main.yml:52]: That's handled inside the template.

**Comment by fredsmith** [comment]
👍  LGTM

**Comment by fredsmith** [comment]
👍

**Pull request merged**